plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -616,9 +615,6 @@ private boolean onlyOnePluginIncluded() {
         top.put("id", DEFAULT_SOURCE_ID);
         JSONObject plugins = new JSONObject();
         try (JarFile jf = new JarFile(war)) {
-            if (pluginGroupIds == null) {
-                pluginGroupIds = new HashMap<>();
-            }
```

Comment:
This was dead code. We were always passing in a non-`null` value for `pluginGroupIds`.

Reply:
I suggest adding annotation to the argument to indicate that